files/shorewall

```diff
@@ -0,0 +1,5 @@
+SHOREWALL_TCP .*?%{WORD:fwrule}\:%{WORD:fwtarget}\:IN\=%{WORD:fwin} OUT\=\s*MAC\=%{DATA:fwmac} SRC\=%{IPORHOST:fwsrc} DST\=%{IPORHOST:fwdst} .*? PROTO\=%{WORD:fwproto} SPT\=%{INT:fwspt:int} DPT\=%{INT:fwdpt:int} .*?
```

Comment:
btw I found why MAC type is not working, this is not a real mac, shorewall is just joining 2 macs, the origin and destination :smile: